### PR TITLE
Increase unit test coverage

### PR DESCRIPTION
**Pull request by timja-kainos**
Removed some code we aren't using

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/cmc/claimstore/processors/JsonMapperTest.java [hunk 1]
```diff
@@ -113,4 +116,15 @@ public void shouldProcessDependantResponseFromJson() throws Exception {
         assertThat(output).isEqualTo(expected);
     }
 
+    @Test(expected = InvalidApplicationException.class)
+    public void shouldThrowExceptionOnInvalidJson() {
+        processor.fromJson("{asads:", ResponseData.class);
+    }
+
+    @Test(expected = InvalidApplicationException.class)
+    public void shouldThrowExceptionOnInvalidJsonWithTypeReference() {
```
Comment (rlewan) [inline src/test/java/uk/gov/hmcts/cmc/claimstore/processors/JsonMapperTest.java]: Does this test add anything beside the first one?

File: src/test/java/uk/gov/hmcts/cmc/claimstore/utils/PartyUtilsTest.java [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+            .isEqualTo(PartyUtils.ON_BEHALF_OF_AN_ORGANISATION);
+
+        assertThat(PartyUtils.getType(SampleTheirDetails.builder().soleTraderDetails()))
+            .isEqualTo(PartyUtils.SOLE_TRADER_OR_SELF_EMPLOYED_PERSON);
+
+    }
+
+    @Test
+    public void getDefendantContactPersonReturnsContactPerson() {
+        CompanyDetails defendant = SampleTheirDetails.builder().companyDetails();
```
Comment (rlewan) [inline src/test/java/uk/gov/hmcts/cmc/claimstore/utils/PartyUtilsTest.java]: Change variable type to `TheirDetails` to make the test more readable?

Comment (timja-kainos) [reply src/test/java/uk/gov/hmcts/cmc/claimstore/utils/PartyUtilsTest.java]: it accesses the contactPerson one line down which is only available on CompanyDetails

File: src/test/java/uk/gov/hmcts/cmc/claimstore/processors/JsonMapperTest.java (hunk 1 above)
Comment (timja-kainos) [reply src/test/java/uk/gov/hmcts/cmc/claimstore/processors/JsonMapperTest.java]: Calls a different overloaded method

File: src/test/java/uk/gov/hmcts/cmc/claimstore/utils/PartyUtilsTest.java [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+import uk.gov.hmcts.cmc.claimstore.models.sampledata.SampleTheirDetails;
+
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class PartyUtilsTest {
+
+    @Test
+    public void getType() {
```
Comment (rlewan) [inline src/test/java/uk/gov/hmcts/cmc/claimstore/utils/PartyUtilsTest.java]: `getTypeMapsPartyTypesCorrectly`

File: src/test/java/uk/gov/hmcts/cmc/claimstore/utils/PartyUtilsTest.java [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+        assertThat(PartyUtils.getType(SampleParty.builder().organisation()))
+            .isEqualTo(PartyUtils.ON_BEHALF_OF_AN_ORGANISATION);
+
+        assertThat(PartyUtils.getType(SampleParty.builder().soleTrader()))
+            .isEqualTo(PartyUtils.SOLE_TRADER_OR_SELF_EMPLOYED_PERSON);
+    }
+
+    @Test
+    public void getTypeTheirDetails() {
```
Comment (rlewan) [inline src/test/java/uk/gov/hmcts/cmc/claimstore/utils/PartyUtilsTest.java]: getTypeMapsTheirDetailsTypesCorrectly

**Pull request merged**